src/server/index.js

```diff
@@ -66,6 +71,30 @@ if (program.host) {
 }
 
 const app = express();
+let server = app;
+
+if (program.https) {
+  if (!program.sslCa) {
+    logger.error('Error: --ssl-ca is required with --https');
+    process.exit(-1);
+  }
+  if (!program.sslCert) {
+    logger.error('Error: --ssl-cert is required with --https');
+    process.exit(-1);
+  }
+  if (!program.sslKey) {
+    logger.error('Error: --ssl-key is required with --https');
+    process.exit(-1);
+  }
+
+  const sslOptions = {
+    ca: fs.readFileSync(program.sslCa, 'utf-8'),
```

Comment:
@doctyper looking at the docs, it seems like `ca` should be an array, and also is only used for self-signed certificates?

https://nodejs.org/api/https.html
https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener

If it's straightforward can you provide simple instructions for me to test this?